install/python-installer/db_uninstall.py

```diff
@@ -96,13 +96,13 @@ def main():
 
     remotes = [Remote(node, pwd=pwd) for node in nodes]
     # stop trafodion on the first node
-    remotes[0].execute('sudo su %s -l -c ckillall' % TRAF_USER, chkerr=False)
+    #remotes[0].execute('sudo su %s -l -c ckillall' % TRAF_USER, chkerr=False)
 
     # remove trafodion userid and group on all trafodion nodes, together with folders
     for remote in remotes:
         info('Remove Trafodion on node [%s] ...' % remote.host)
         remote.execute('ps -ef|grep ^%s|awk \'{print $2}\'|xargs sudo kill -9' % TRAF_USER, chkerr=False)
-        remote.execute('ps -ef|grep krb5check|awk \'{print $2}\'|xargs sudo kill -9', chkerr=False) # traf kerberos service
+        remote.execute('trafid=`getent passwd %s|awk -F: \'{print $3}\'`; if [[ -n $trafid ]]; then ps -ef|grep ^$trafid|awk \'{print $2}\'|xargs sudo kill -9; fi' % TRAF_USER, chkerr=False)
         remote.execute('sudo -n /usr/sbin/userdel -rf %s' % TRAF_USER, chkerr=False)
```

Comment:
on lines 104,105, instead of doing ps -ef and grep for TRAF_USER, wouldn't it be better to do 

'ps -f -u %s' % TRAF_USER